.github/workflows/update-test-times.yml

```
@@ -23,14 +23,22 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Set up python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+
       - name: Install Dependencies
-        run: pip3 install boto3==1.19.12 rockset==1.0.3
+        run: python -m pip install boto3==1.19.12 clickhouse-connect==0.7.16 requests==2.26.0
```

Can you please explain why changing `python` to `python3` is necessary?
```suggestion
        run: python3 -m pip install boto3==1.19.12 clickhouse-connect==0.7.16 requests==2.26.0
```

---

I switched from python3 to python since I added the setup python step, which makes the default python the version that gets setup in the setup step